getNodeById/function.json

```
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "name": "getNodeById",
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "methods": ["get"],
+      "route": "node/{id}"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "$return"
+    }
+  ],
+  "scriptFile": "../dist/getNodeById/index.js"
```

```suggestion
  "scriptFile": "../dist/GetNodeById/index.js"
```

Change folder's name as well. (To do so, commit twice- one without the folder, and one with the uppercase name)

---

You need to change folders names as well (you can see that in the git files the name is not updated)